var/spack/repos/builtin/packages/chill/package.py

```diff
@@ -19,15 +19,16 @@ class Chill(AutotoolsPackage):
     version('0.3', sha256='574b622368a6bfaadbe9c1fa02fabefdc6c006069246f67d299f943b7e1d8aa3')
 
     depends_on('boost@1.66.0 cxxstd=11', type='build')
-    depends_on('rose@0.9.10.0 +cxx11', type='build')
+    depends_on('rose@0.9.13.0: +cxx11', type=('build', 'run'))
     depends_on('autoconf', type='build')
     depends_on('automake@1.14:',  type='build')
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
-    depends_on('iegenlib', type='build')
-    depends_on('bison@3.4', type='build')
+    depends_on('iegenlib', type=('build', 'run'))
+    depends_on('bison@3.4.2:', type='build')
     depends_on('flex', type='build')
-    depends_on('python')
+    # Does not currrently work with Python3
+    depends_on('python@2.7:2.8')
```

Comment:
When is Python actually used? Is it used a build-time? Run-time? Do we link to the Python libraries?

Reply:
You can build CHiLL to include a Python interpreter. It includes `-lpython2.7` in `Makefile.am`, which tells me Python is necessary and build, link, and runtime.

Reply:
In that case you'll want:
```suggestion
    depends_on('python@2.7:2.8', type=('build', 'link', 'run'))
```
This will add the executable to the `PATH` during build-time and run-time, and add the libs/headers via the compiler wrappers at build-time.

Reply:
The CHiLL runtime does not actually launch the python executable, but rather loads the Python interpreter via libpython2.7.so. This means Python should be installed at runtime, but does not need `PATH` to be set to the python interpreter executable. So far everything runs fine for me without include `type=(..., 'run')` in `depends_on`.

I'm happy to make the change if you feel it is necessary, but I have not experienced any problems with CHiLL installed without the run build type on python.

Reply:
If it doesn’t to be in the PATH at run-time, it sounds like the default build/link is correct